Assets/MixedRealityToolkit/_Core/Devices/BaseController.cs

```diff
@@ -84,12 +85,19 @@ protected IMixedRealityInputSystem InputSystem
         /// <param name="controllerType"></param>
         public void SetupConfiguration(Type controllerType)
         {
+
             if (MixedRealityManager.Instance.ActiveProfile.IsControllerMappingEnabled)
             {
+                if (MixedRealityManager.Instance.ActiveProfile.ControllerMappingProfile.RenderMotionControllers)
+                {
+                    TryRenderControllerModel(controllerType);
```

Comment:
Good that we are bringing this back in to the base controller.
However, it only covers creation, what about destruction of the controller when source is lost?

Reply:
Destruction is handled in the pose synchronizer

Reply:
Now that I look back the interface already has you implement `DestroyOnSourceLost` to satisfy the requirements of it handling destruction of the object

Reply:
OK,  I missed the requirement that this implementation required prefabs, so that should be noted in the documentation

Reply:
Added doc comments in interface.